src/app/shared-module/services/task.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { Task } from '../types/task';
+import { Task } from '../../home-page/types/task';
```

Comment:
Maybe `Task` should also be in the shared module?

Reply:
Done